java/src/main/java/io/vertx/ext/eventbus/client/options/ProxyOptions.java

```diff
@@ -0,0 +1,177 @@
+package io.vertx.ext.eventbus.client.options;
+
+import java.util.Objects;
+
+/**
+ * @author <a href="mailto:pl@linux.com">Phil Lehmann</a>
```

Comment:
are you the brother of Alex btw :-) ?

Reply:
Nooo, where I come from every second guy is called Lehmann ;)